src/Tizen.NUI/src/public/Common/Layer.cs

```diff
@@ -341,6 +341,14 @@ public override void Remove(View child)
             {
                 throw new ArgumentNullException(nameof(child));
             }
+            if (child.GetParent() == null) // Early out if child parent is null.
+                return;
+
+            if (child.GetParent() != this)
+            {
+                //throw new System.InvalidOperationException("You have deleted a view that is not a child of this layer.");
+                Tizen.Log.Error("NUI", "You have deleted a view that is not a child of this layer.");
```

Comment:
I think this should return at this point after printing Error log.
Because if child's parent is not this, then child cannot exist in Children.
Moreover, I think that the following codes should not be called if child is not a child object.
```
            child.InternalParent = null;
            LayoutCount -= child.LayoutCount;
```

Reply:
I also catch that points.
But it is not a kind of TCT fix-up patches So I had just plan to do that jobs with another PR.

I saw your PR. it looks god to me, So I also fix it at this PR.